File: src/test/java/com/dotnomi/fabricdependencyinjection/java/testclasses/success/SuccessModMain.java
package com.dotnomi.fabricdependencyinjection.java.testclasses.success;

import com.dotnomi.fabricdependencyinjection.annotation.ModInject;
import com.dotnomi.fabricdependencyinjection.annotation.ModMain;

@ModMain
public class SuccessModMain {
  @ModInject
  public ServiceA serviceA;
}

File: src/test/java/com/dotnomi/fabricdependencyinjection/kotlin/testclasses/success/SuccessModMain.kt
package com.dotnomi.fabricdependencyinjection.kotlin.testclasses.success

import com.dotnomi.fabricdependencyinjection.annotation.ModInject
import com.dotnomi.fabricdependencyinjection.annotation.ModMain

@ModMain
class SuccessModMain {
    @ModInject
    lateinit var serviceA: ServiceA
}
